Topic 1/src/point7/Music.java

```diff
@@ -0,0 +1,78 @@
+package point7;
+
+public class Music {
+	private String disk; // indicates which disk or album the song belongs
+	private String artist;
+	private String title;
+	private int duration;
+	private boolean state; // indicates whether this on or off
+	
+	public Music(String disk, String artist, String title, int duration){
+		this.setDisk(disk);
+		this.setArtist(artist);
+		this.setTitle(title);
+		this.setDuration(duration);
+		this.setState(false);
+	}
+
+	public boolean equals (Object o) {
```

Comment:
[Medium] if you are overriding a method, it is a good practice to include <code>@Override</code>. You can read about it in: http://docs.oracle.com/javase/7/docs/api/java/lang/Override.html